ui/js/models/repository.js

```diff
@@ -54,17 +82,42 @@ treeherder.factory('ThRepositoryModel', [
             // The $interval will pass in the number of times it was called,
             // rather than a ``repoName``.  So repoName would equal 1, 2, 3.  So
             // if repoName isn't a valid watched repo, we update all.
-            let repoNames = watchedRepos.includes(repoName) ? [repoName] : watchedRepos;
+            let repoNames = watchedRepos.indexOf(repoName) !== -1 ? [repoName] : watchedRepos;
 
             // filter out non-watched and unsupported repos to prevent repeatedly
             // hitting an endpoint we know will never work.
             repoNames = repoNames.filter(repo => watchedRepos.indexOf(repo) !== -1 && repos[repo].treeStatus.status !== 'unsupported');
 
-            repoNames.forEach((repoName) => {
-              TreeStatusModel.get(repoName).then((data) => {
-                repos[repoName].treeStatus = data.result;
-              });
-            });
+            const newStatuses = {};
```

Comment:
It looks like the manual rebase has added back some code that was removed in another PR. Could you remove it?

Reply:
Sure :)